### PR TITLE
exui-2220-add logs

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EXUI-2220

### Change description ###
add logs

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by andywilkinshmcts — APPROVED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts [hunk 1]
```diff
@@ -221,6 +226,7 @@ export class WorkAllocationService {
       tasks: []
     };
     if (!this.isWAEnabled()) {
+      this.appConfig.logMessage(`search for completable call missed for ${caseId} in event ${eventId}`);
```
Comment (andywilkinshmcts) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts:229]: Can you change this message to "isWAEnabled false for ${caseId} in event {eventId}"

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts [hunk 2]
```diff
@@ -54,7 +54,8 @@ export class WorkAllocationService {
   private isWAEnabled(jurisdiction?: string, caseType?: string): boolean {
     this.features = this.appConfig.getWAServiceConfig();
     const ftstr = JSON.stringify(this.features);
-    console.log(`isWAEnabled: ${ftstr}`)
+    console.log(`isWAEnabled: ${ftstr}`);
```
Comment (andywilkinshmcts) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts:57]: You could remove the console.log

**Review by andywilkinshmcts — DISMISSED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts (hunk 2 above)
Comment (RiteshHMCTS) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts]: done

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "7.0.60",
+  "version": "7.0.58-exui-2220-rc4",
```
Comment (andywilkinshmcts) [inline package.json:3]: Version number needs to be updated to 7.0.61

**Review by andywilkinshmcts — CHANGES_REQUESTED** [review]

**Review by Josh-HMCTS — APPROVED** [review]

**Review by Josh-HMCTS — APPROVED** [review]

**Pull request merged**